tests/utls/context-mock.ts

```diff
@@ -0,0 +1,12 @@
+import {contextService} from "../../src/services/context.service";
+
+export function setDefaultProfile(): void {
+    contextService.setContext({
+        profile: {
+            name: "test",
+            team: "https://myTeam.celonis.cloud/",
+            apiToken: "YnQ3N2M0M2ItYzQ3OS00YzgyLTg0ODgtOWNkNzhiNzYwOTU2OlFkNnBpVCs0M0JBYm1ZWGlCZ2hPd245aldwWTNubFQyYVFOTFBUeHEwdUxM",
```

Comment:
I don't think we need a real token here :smile:.

Reply:
This is a non-valid mocked token anyway.